portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -87,13 +87,66 @@ definition {
 	}
 
 	@description = "Verify that a Color Field can be set to repeatable"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeSetToRepeatable {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133136 CanBeSetToRepeatable pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+		
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+		
+		Form.gotoAdvancedTab();
+		
+		FormViewBuilder.setFieldRepeatable();
+
+		AssertElementPresent(
+			key_fieldName = "Color",
+			key_repeatableFieldNumber = "1",
+			locator1 = "FormFields#FIELD_ADD_SPECIFIC");
+		
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.addRepeatableField(
+			fieldName = "Color",
+			repeatableFieldNumber = "1");
+
+		FormFields.viewColorValueLabelPickerByRow(
+			fieldLabel = "Color",
+			fieldName = "Color",
+			rowNumber = "2",
+			colorValue = "000000");
```

Comment:
@lima-rafa, can you please rename this macro to `viewColorField` only?

Additionally, do you really need the both parameters `fieldLabel` and `fieldName` considering they have the same value?

From what I can see, we just need the parameter `fieldLabel` and inside your macro you can reuse it. :wink:

Reply:
Rodrigo I believe not, one thing is the "Label" value (which represents a simple text that appears on the screen) and the other is the "Name" (which represents a value of an HTML attribute and does not appear on the screen)

For example a test "single selection" its "Label" value is "Single Selection" or it can be any other text, but "Name" would have the value "SingleSelection" the two are not equal.

Reply:
Understood, @lima-rafa